openapi.yml

```diff
@@ -5875,6 +6169,11 @@ components:
       - JIRA
       - BUGZILLA
       type: string
+    Type154Enum:
```

Comment:
I was discussion with Marian that they would like to also have a type `REPRODUCER` and `PATCH`. Has he reached out to you already? That should be pretty small change and it would cover maybe 10 additional acceptance test cases.

Reply:
He told me about it this week, but I would prefer to create a new Jira with a detailed description (e.g. if we need validations etc.).

Reply:
I do not have any preferences on how to track it. It would be good to deliver it as part of this initiative (can be a different PR, does not matter) so we can claim the reference functionality done as a whole.

Reply:
I asked Marian to create a Jira with all details, so we can discuss it later.

Reply:
The reproducer is definitely an internal thing so it will never go to the CVE pages or so. I am not sure about the patch. The purpose should be described by IR team. There are already those types of links being added probably often but just as the external type.

Reply:
My question is basically "will anyone actually use this?" we probably don't even need a reference type, I am sure SDEngine could tell which one is a security bulletin or not just by looking at the URL

Reply:
We do not know whether and how is this going to use. IR team would like to have it and it is a little work to add it so I do not think this is a big deal. If it was a new functionality then the reasoning should be more precise.

Maybe SDEngine could recognize them but 1) the restriction that it begins with `https://access.redhat.com/` does not ensure it (CVE pages and much more is also there) 2) the location may change which should not make the historical articles (or other references) not being articles any more 3) the restriction is not even enforcing and based on analysis Jitka did the actual data are wild west so auto-recognition would be unreliable.